arithmetization/src/test/java/net/consensys/linea/zktracer/module/hub/signals/WarmingScenarii.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright ConsenSys Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package net.consensys.linea.zktracer.module.hub.signals;
+
+public enum WarmingScenarii {
+    NO_WARMING,
+    WARMING_SENDER,
+    WARMING_RECIPIENT,
+    WARMING_COINBASE,
+    WARMING_PRECOMPILE,
+    WARMING_TO_BE_DEPLOYED_STORAGE
```

Comment:
I take it that this would be a scenario to warm a storage key `key` and the associated address (which is the address of a deployment) `depAddr`, but there are two possible ways to do this:
- `depAddr` is generated in a deployment transaction
- `depAddr` is generated in a `CREATE(2)`